data_fetcher.py

```diff
@@ -0,0 +1,110 @@
+import os
+import shutil
+import json
+import pandas as pd
+import yfinance as yf
+from datetime import datetime
+
+class DataFetcher:
+    """
+    This module downloads data from yahoo finance and saves it locally in a downloads directory
+    """
+    def __init__(self, directory_path: str, import_start_date: datetime=datetime(2008,1,1)) -> None:
+        self.module_version = 1
```

Comment:
Should we add this parameter to other moduls? Additinally, I think it should be a string and starting the count from 0.1.0, and then 0.2.0 etc.

Reply:
ok, the best practice here is to use __version__
I will change this